onebusaway-android/src/main/res/layout/arrivals_list_header_row_template.xml

```diff
@@ -2,145 +2,107 @@
+                android:layout_toRightOf="@id/eta_route_name"
+                android:layout_alignBaseline="@id/eta_route_name"
+                android:gravity="bottom"/>
+    </RelativeLayout>
 
-        <RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
-                        android:id="@+id/eta_and_min"
-                        android:layout_width="wrap_content"
-                        android:layout_height="wrap_content"
-                        android:layout_column="2"
-                        android:layout_marginLeft="@dimen/arrival_header_text_pad_left"
-                        android:layout_marginTop="3dp"
-                        android:layout_marginBottom="3dp"
-                        android:paddingLeft="4dp"
-                        android:paddingRight="4dp"
-                        android:gravity="right"
-                        android:orientation="horizontal"
-                        android:background="@drawable/round_corners_style_b_header_status">
-                <TextView
-                        android:id="@+id/eta"
-                        android:layout_width="wrap_content"
-                        android:layout_height="wrap_content"
-                        android:text="2"
-                        android:textColor="@color/header_text_color"
-                        android:textStyle="bold"
-                        android:ellipsize="end"
-                        android:singleLine="true"/>
-                <TextView
-                        android:id="@+id/eta_min"
-                        android:layout_width="wrap_content"
-                        android:layout_height="wrap_content"
-                        android:textSize="14sp"
-                        android:text="min"
-                        android:textColor="@color/header_text_faded_color"
-                        android:ellipsize="end"
-                        android:singleLine="true"
-                        android:textIsSelectable="false"
-                        android:layout_marginLeft="5dp"
-                        android:layout_toRightOf="@id/eta"
-                        android:layout_alignBaseline="@id/eta"
-                        android:gravity="bottom"/>
-                <include
-                        android:id="@+id/eta_realtime_indicator"
-                        layout="@layout/realtime_indicator"
-                        android:layout_width="6dp"
-                        android:layout_height="6dp"
-                        android:layout_toRightOf="@id/eta_min"
-                        android:layout_marginTop="16dp"
-                        android:layout_marginLeft="1dp"/>
-        </RelativeLayout>
+    <include
+            android:id="@+id/eta_and_min"
+            layout="@layout/eta_header_view"
+            android:layout_marginTop="3dp"
+            android:layout_marginBottom="3dp"
+            android:layout_height="wrap_content"
+            android:layout_width="wrap_content"/>
```

Comment:
I think you might need to add these to the legend layout as well.  It looks like the padding on the left is less for the scheduled time than it is for real-time arrivals.